.github/workflows/post_release.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       - name: Generate Docs Site
         run: $(xcrun --find docc) process-archive transform-for-static-hosting ios/Example/Pods/SmileID.doccarchive --output-path docs --hosting-base-path ios
       - name: Push gh-pages branch
-        uses: stefanzweifel/git-auto-commit-action@v5
+        uses: stefanzweifel/git-auto-commit-action@v6
```

Comment:
@tobitech given that we are not using any of the breaking changes in  https://github.com/stefanzweifel/git-auto-commit-action/releases/tag/v6.0.0 , we should be fine to merge this?